lib/pavilion/series/test_set.py

```diff
@@ -722,14 +725,22 @@ def wait(self, wait_for_all=False, wait_period: int = TEST_WAIT_PERIOD) -> int:
         :return: The number of completed tests
         """
 
+        timeout = set_default(timeout, math.inf)
+
         # No tests to wait for
         if not self.started_tests:
             return 0
 
         completed_tests = self.mark_completed()
 
+        start = time.monotonic()
+
```

Comment:
Using the monotonic clock really seems like over kill.  I'd like to hear why it matters.

Reply:
The choice was motivated by this note in the docs for `time.time`:

> Note that even though the time is always returned as a floating point number, not all systems provide time with a better precision than 1 second. While this function normally returns non-decreasing values, it can return a lower value than a previous call if the system clock has been set back between the two calls.